src/Mahi/Util/Console.cpp

```diff
@@ -342,17 +342,17 @@ int get_key(void) {
         case 0: {
             int kk;
             switch (kk = getch()) {
-                case 71: return KEY_NUMPAD7;
-                case 72: return KEY_NUMPAD8;
-                case 73: return KEY_NUMPAD9;
-                case 75: return KEY_NUMPAD4;
-                case 77: return KEY_NUMPAD6;
-                case 79: return KEY_NUMPAD1;
-                case 80: return KEY_NUMPAD2;
-                case 81: return KEY_NUMPAD3;
-                case 82: return KEY_NUMPAD0;
-                case 83: return KEY_NUMDEL;
-                default: return kk-59+KEY_F1; // Function keys
+                case 71: return KEY_HOME;
+                case 72: return KEY_UP;
+                case 73: return KEY_PGUP;
+                case 75: return KEY_LEFT;
+                case 77: return KEY_RIGHT;
+                case 79: return KEY_END;
+                case 80: return KEY_DOWN;
+                case 81: return KEY_PGDOWN;
+                case 82: return KEY_INSERT;
+                case 83: return KEY_DELETE;
+                default: return kk-59 + KEY_F1; // Function keys
```

Comment:
Would this line be a problem do to the new defin of KEY_F1?

Reply:
This is just shorthand for not needing to write down all cases as
```cpp
case 59: return KEY_F1;
case 60: return KEY_F2;
case 61: return KEY_F3;
case 62: return KEY_F4;
// etc...
```
Although, I just realized that with the current setup, the `kk-59+KEY_F1` is redundant now that `KEY_F1` is the 59 enum, so this essentially evaluates to `kk`.